### ISSUE TITLE
consulta query 5

### ISSUE THREAD

**Issue by mbernasconi04** [PI-ITBA/2024_01#447]
solo se consideran las infracciones que tuvieron multas entre las fechas que nos indican los argumentos no? o se considera que la diferencia entre el monto máximo y el mínimo es 0, si no hubo multas entre esas fechas.

**Comment by marcelogarberoglio** [comment]
Sí, sólo se consideran las que están entre los dos años que pasaron por argumento
